Makefile

```diff
@@ -68,16 +68,16 @@ clean:
 
 .PHONY: install-tools
 install-tools:
-	go install github.com/google/go-licenses@v1.0.0
+	go install github.com/google/go-licenses/v2@v2.0.0-alpha.1
```

Comment:
Do we want to use an alpha version here? Also wondering if we should separate this into an env var further up so it's easier to spot and change (similar to the dockerize versions), vs in the middle of the command. No strong opinion here though.

Reply:
This tool has been slow to evolve with many changes in the 1.x versions; we were stuck on 1.0 because of problems with later versions. I'd personally rather see us standardize on 2.x even if it's alpha.

Reply:
I don't think there would be a need to dynamically update the liceses version and as Kent said, they release versions *very slowly*. Let's go with this as is for now, and add the env var later if needed.